tests/SprykerEcoTest/Zed/Braintree/Business/Order/SaverTest.php

```diff
@@ -254,16 +284,20 @@ protected function createAndGetAddressEntity(SpyCountry $countryEntity)
      */
     protected function createAndGetCustomerEntity()
     {
-        $customerEntity = (new SpyCustomer())
-            ->setFirstName('John')
-            ->setLastName('Doe')
-            ->setEmail('john@doe.com')
-            ->setDateOfBirth('1970-01-01')
-            ->setGender(SpyCustomerTableMap::COL_GENDER_MALE)
-            ->setCustomerReference('braintree-pre-authorization-test');
-        $customerEntity->save();
+        $customer = (new SpyCustomerQuery())
```

Comment:
why did you change correct `$customerEntity` naming to this one?

Reply:
Fixed the naming